src/Operator/Constructors.jl

```diff
@@ -763,7 +861,8 @@ operator field:
     gradient
 ```
 """
-function advectionoperator(basis::AbstractBasis, mesh::Mesh, wind = [1, 1])
+function advectionoperator(basis::AbstractBasis, mesh::Mesh)
+    wind = [1, 1]
```

Comment:
Should wind be an argument? It feels like something we shouldn't hard-code

Reply:
Alright, I will put it back as argument. It was actually sitting there before.